_sass/theme/_colors.scss

```diff
@@ -0,0 +1,9 @@
+$canvas-base: #ffffff;
+$content-base: #000000;
+$utility-theme-color: $primary-color;
```

Comment:
Where does this come from?

Reply:
this comes from our `styles.scss` - we have to use `import` sadly due to the version of `sass-embedded` not supporting `use`. upgrading it causes errors on m1 mac, which i'm abit scared to do (fixed on local) given the age of the script